docs/release-notes/RELEASE-NOTES-2.2.0.md

```diff
@@ -1,14 +1,32 @@
 # About this release
 
 [This][this-release] is a maintenance release.
-It:
+
+It finalized renaming process of the software.
+
+Please refer to the notes below for details and backward compatibility.
+
+Regarding renaming, this release:
+* Changes Erlang node name from `epoch@localhost` to `aeternity@localhost`. This impacts persisted database (the node name is stored in it). **In order to use persisted database created before this release, new tool - `rename_db` - must be used.**
+  * The tool `rename_db` is included in the release. It takes one argument - path to database directory. Path to your database directory is `chain` > `db_path` parameter in your user config. If it is undefined, `data` (default directory) should be passed as argument.
+  * Example usage of the tool:
+    * Using absolute path of database directory - `./bin/aeternity rename_db /node/aeternity/node/my-old-db-path`;
+    * Using relative path of database directory (or when `chain` > `db_path` is not set in the config) - `./bin/aeternity rename_db data`;
+    * If you are running a node using Docker (assuming you either don't have `db_path` set in your config, or it is set to `/home/aeternity/node/data`) - `docker run --entrypoint=/bin/bash -v ~/.aeternity/myaedb:/home/aeternity/node/data/mnesia -v ~/.aeternity/myaeternity.yaml:/home/aeternity/.aeternity/aeternity/aeternity.yaml aeternity/aeternity -c "/home/aeternity/node/bin/aeternity rename_db /home/aeternity/node/data"`
```

Comment:
Why not entrypoint the bin/aeternity script? (Untested.)
```suggestion
    * If you are running a node using Docker (assuming you either don't have `db_path` set in your config, or it is set to `/home/aeternity/node/data`) - `docker run --entrypoint=/home/aeternity/node/bin/aeternity -v ~/.aeternity/myaedb:/home/aeternity/node/data/mnesia -v ~/.aeternity/myaeternity.yaml:/home/aeternity/.aeternity/aeternity/aeternity.yaml aeternity/aeternity rename_db /home/aeternity/node/data`
```

Reply:
@zp-sd Can you please confirm that `bash -c ...` does not lose exit code? If it loses exit code, I think this doc has to change.

Reply:
It seems it keeps exit code 🙂 .

```
➜  aeternity git:(pt-163423623-rename-mnesia-db-3) ✗ docker run --entrypoint=/bin/bash -v /Users/zp/src/aeternity/test-db/mnesia:/home/aeternity/node/data/mnesia -v ~/.aeternity/aeternity/aeternity.yaml:/home/aeternity/.aeternity/aeternity/aeternity.yaml aeternity/aeternity:local -c "/home/aeternity/node/bin/aeternity rename_db /home/aeternity/node/data"
Starting node name renaming in the db from "epoch@localhost" to "aeternity@localhost"
Created schema.DAT backup file at: "/home/aeternity/node/data/mnesia/schema.DAT.backup". Please restore from it if renaming process is interrupted or fails.
➜  aeternity git:(pt-163423623-rename-mnesia-db-3) ✗ echo $?
0
```

```
➜  aeternity git:(pt-163423623-rename-mnesia-db-3) ✗ docker run --entrypoint=/bin/bash -v /Users/zp/src/aeternity/test-db/mnesia:/home/aeternity/node/data/mnesia -v ~/.aeternity/aeternity/aeternity.yaml:/home/aeternity/.aeternity/aeternity/aeternity.yaml aeternity/aeternity:local -c "/home/aeternity/node/bin/aeternity rename_db /home/aeternity/node/foobar"
Starting node name renaming in the db from "epoch@localhost" to "aeternity@localhost"
Schema file not found: "/home/aeternity/node/foobar/mnesia/schema.DAT"
Script changing node in the db failed: {error,file_not_found}
➜  aeternity git:(pt-163423623-rename-mnesia-db-3) ✗ echo $?
1
```